### PR TITLE
DOC: Accessing files from a S3 bucket.

### LINKED ISSUE #12206: DOC: improve s3 access doc-strings / docs
https://github.com/pydata/pandas/pull/12198

### PR DESCRIPTION
**Pull request by myles**
Add some documentation about accessing files from a remote S3 bucket in pandas.

- [x] closes #12206
- [x] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/cookbook.rst [hunk 1]
```diff
@@ -1370,3 +1370,28 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+
+Load a file from S3
+-------------------
+
+Pandas support loading files from a S3 bucket for remote file interactivity.
```
Comment (alimcmaster1) [inline doc/source/cookbook.rst]: NIT: But I think this should read 'Pandas supports loading files'

**Review by alimcmaster1 — CHANGES_REQUESTED** [review]

**Comment by datapythonista** [comment]
We already have this: https://pandas.pydata.org/pandas-docs/stable/io.html#reading-remote-files

I'd expand that section if there is anything missing, I wouldn't expect many people to look in the cookbook for S3 options having a page specific for IO operations.

Does it make sense?

**Comment by myles** [comment]
@datapythonista I've expanded the the section in io and remove it from cookbook.

**Review by datapythonista — APPROVED** [review]
Thanks for updating @myles. That looks good to me, but I don't know much about S3, so will let other people review.

**Comment by jreback** [comment]
thanks!

**Pull request merged**